.ci/openshift-ci/cluster/deployments/daemonset_kata-installer.yaml.in

```diff
@@ -35,6 +35,12 @@ spec:
                   valueFrom:
                   fieldRef:
                    fieldPath: spec.nodeName
+                - name: QEMU_PATH
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ci.kata.sandboxedcontainers
```

Comment:
fyi: just realized `name` should be `ci.kata.installer`. I will fix it.

Reply:
Fixed. I am actually now using two config maps: `ci.kata.installer.qemu`  and `ci.kata.installer.kernel` as I realized updating a configMap is a non-trivial task.